client/src/modules/Sidebar/components/ItemList/components/DefItem/EditDefItem/index.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase */
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import MyForm from '../../../../../../../shared/Form';
+import EditInRedux from './EditInRedux';
+import EditInDB from './EditInDB';
+import { editDefPoint } from '../../../actions/list';
```

Comment:
Use an action editDefItem from src\modules\Sidebar\components\ItemList\actions\list.js instead. It does the same edit on the front and server side.